packages/webapp/esbuild.js

```diff
@@ -17,7 +17,7 @@ require('esbuild')
         format: 'cjs',
         bundle: true,
         metafile: true,
-        sourcemap: true, // .vscodeignore ignores .map files when bundling!!
+        sourcemap: process.argv.includes('--watch') ? 'inline' : true, // .vscodeignore ignores .map files when bundling!!
```

Comment:
🚀

Reply:
Yes, but there is still an issue when I try to set `minify` to `false` for watch, which might be solved with https://github.com/SAP/guided-answers-extension/issues/364 and your PR https://github.com/SAP/guided-answers-extension/pull/384

Reply:
Issue with `minify` is fixed, checked after https://github.com/SAP/guided-answers-extension/pull/384 was merged